### PR TITLE
Rs/atlas

### PR DESCRIPTION
**Pull request by ravishetye**

### CODE REVIEW

File: verta/verta/client.py [hunk 1]
```diff
@@ -799,6 +815,85 @@ def __init__(self, job_id=None, query="", execution_timestamp="",
...
+            self.attributes = self.get_attributes(table_obj)
+            self.tags = self.get_tags(table_obj)
+        else:
+            super(AtlasHiveDatasetVersionInfo, self).__init__()
+
+    @staticmethod
+    def get_tags(table_obj):
+        verta_tags = []
+        if 'classifications' in table_obj:
+            atlas_classifcations = table_obj['classifications']
```
Comment (convoliution) [inline verta/verta/client.py]: "classifcation" is missing an 'i'

File: verta/verta/client.py [hunk 2]
```diff
@@ -799,6 +815,85 @@ def __init__(self, job_id=None, query="", execution_timestamp="",
...
+            for atlas_classifcation in atlas_classifcations:
```
Comment (convoliution) [inline verta/verta/client.py]: see above

File: verta/verta/client.py [hunk 3]
```diff
@@ -799,6 +815,85 @@ def __init__(self, job_id=None, query="", execution_timestamp="",
...
+    def get_attributes(table_obj):
+        attribute_keyvals=[]
+        attributes = {}
+        attributes['type'] = table_obj['typeName'] 
+        attributes['table_name'] = table_obj['attributes']['name'] # store as attribute
+        attributes['database_name'] = table_obj['relationshipAttributes']['db']['displayText'] #store as atrribute
+        attributes['col_names'] =  AtlasHiveDatasetVersionInfo.get_columns(table_obj)
+        attributes['created_time'] =  table_obj['createTime']
+        attributes['update_time'] =  table_obj['updateTime']
+        attributes['load queries'] = AtlasHiveDatasetVersionInfo.get_inbound_queries(table_obj)
```
Comment (convoliution) [inline verta/verta/client.py]: Should we have this be 'load_queries', with the underscore?

File: verta/verta/client.py (hunk 1 above)
Comment (ravishetye) [reply verta/verta/client.py]: Fixed.

File: verta/verta/client.py (hunk 2 above)
Comment (ravishetye) [reply verta/verta/client.py]: as above

File: verta/verta/client.py (hunk 3 above)
Comment (ravishetye) [reply verta/verta/client.py]: load_queries it is

**Pull request merged**